RecoLocalCalo/EcalRecProducers/python/ecalMultiFitUncalibRecHit_cfi.py

```diff
@@ -63,3 +63,6 @@
       chi2ThreshEE_ = cms.double(50.0),
    )                                           
 )
+
+from Configuration.StandardSequences.Eras import eras
+eras.phase2_common.toModify( ecalMultiFitUncalibRecHit.algoPSet, useLumiInfoRunHeader = cms.bool(False) )
```

Comment:
The underlying problem is that bunchSpacingProducer is separate from localreco:
https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/Configuration/StandardSequences/python/Reconstruction_cff.py#L101

Without it, there is an exception when running ecalMultiFitUncalibRecHit. The above customization seemed like the easiest way to prevent the exception. I'm open to other solutions.